newIDE/app/src/UI/iconmoon-font.css

```diff
@@ -17,19 +17,23 @@
   font-weight: normal;
   font-variant: normal;
   text-transform: none;
-  line-height: 1;
+  line-height: 0.9;
```

Comment:
why this change ?

Reply:
well after changing the icons i realised the circle which appears when you hover on the icons had an ellipse like shape.

Reply:
That may be due to non square icons. Would be interesting then to add some margin/padding to the `icon-xxx` rules, instead of modifying this because it sounds risky.

Reply:
okay